### DISCUSSION TITLE
Property Principal was not yet loaded

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/pnpcore#323] [Q&A] [answered]
I get one ListItem from a caml query. 
I try to update 2 text fields on it.
```

var itemToUpdate =  myList.Items.GetById(listItem.Id);
 await itemToUpdate.EnsurePropertiesAsync(f => f.Values);
itemToUpdate.Values["Title"] = newECGProject.Name;
itemToUpdate.Values["Practice"] = newECGProject.Practice;
await itemToUpdate.UpdateAsync();

```
I get an error "Property Principal was not yet loaded" 
It seems like in CSOM the load is missing.
I thought itemToUpdate.EnsurePropertiesAsync should do it. 

Please give me an example.
Thank you

**Comment by jansenbe** [comment, marked as answer]
Thanks for reporting this one @Ofer-Gal, it actually turned out to be a bug...it complained about other fields in your list for which no value was set in the update. I've fixed this, please check again with tomorrow's nightly build (version 0.5.32-beta2 or higher)
